lib/parser.js

```diff
@@ -1,15 +1,20 @@
 function _interopDefault(ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex.default : ex; }
-const parser = _interopDefault(require('parse-function'))({ ecmaVersion: 2017 });
+const acorn = require('acorn');
+const parser = _interopDefault(require('parse-function'))({ parse: acorn.parse, ecmaVersion: 11, plugins: ['objectRestSpread'] });
```

Comment:
isn't it the `parse-function` lib that gave us troubles twice in the past?

Reply:
yeah, one of the reason of this PR is to remove it, as I understand